source/dub/generators/build.d

```diff
@@ -546,7 +546,9 @@ class BuildGenerator : ProjectGenerator {
 				auto result = prg_pid.wait();
 				settings.targetExitStatus = result;
 				runPostRunCommands(m_project.rootPackage, m_project, settings, buildsettings);
-				enforce(result == 0, "Program exited with code "~to!string(result));
+				string error = "Program exited with code "~to!string(result);
+				version (Posix) error ~= " (" ~ signalToName(-result) ~ ")";
+				enforce(result == 0, error);
```

Comment:
:facepalm: I just realized I am too eagerly allocating the message. I'll fix it. ._.